Header header
float64 latitude
float64 Longtitude
float64 Altitude
float64 utm_easting
float64 utm_northing
int64 Zone
string letter

